### PR TITLE
feat/registration-confirm-email

### PR DESCRIPTION
**Pull request by AmirAliEidivandi**

### CODE REVIEW

**Comment by ethyaan** [comment]
@AmirAliEidivandi  remove index.html from `emails/html/index.html` and index.mjml `emails/mjml/index.mjml`

File: modules/user/controller.js [hunk 1]
```diff
@@ -40,6 +42,17 @@ class userController {
...
+            const templateTags = [
+                {name: "__USERNAME", value: newUser.email},
+                {name: "__VERIFICATION_CODE", value: verificationCode},
+            ]
+            sendgrid.sendMailByTemplate(
+                'Verification Code',
+                'verification-code',
+                templateTags,
+                [newUser.email],
+                'no-reply@bounce.lingemy.com'
```
Comment (ethyaan) [inline modules/user/controller.js]: change this line to 'no-reply@site.com'

**Review by ethyaan — COMMENTED** [review]
in `services/sendGrid.js` remove below code form `SendMailByTemplate`
```
templateTags.push({
    name: "___ASSET_URL",
    value: this._CONFIG.backendAddress,
});
```

This is wrong code.

**Comment by AmirAliEidivandi** [comment]
ok, fixed

**Review by ethyaan — COMMENTED** [review]
In `sendGrid.js` file, don't  pass any args to the initialized instance of the class.
remove `app`

**Review by ethyaan — APPROVED** [review]

**Pull request merged**